pages/glossary/primary-language-subtag.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Known Primary Language Tag
```

Comment:
Wondering whether this should be "Known Primary Language **sub**tag" instead (so that "tag" is always the full tag, and "subtag" is the hyphen separated bits, staying closer to the RFC.